.github/workflows/dubbo_integration_test.yml

```diff
@@ -69,32 +69,129 @@ jobs:
+            dubbo-versions: '15'
+            registry-model: 'instance'
+          - dubbo-version: '3-1'
+            dubbo-versions: '0'
+            registry-model: 'all'
+          - dubbo-version: '3-1'
+            dubbo-versions: '0'
+            registry-model: 'instance'
+          - dubbo-version: '3-1'
+            dubbo-versions: '3'
+            registry-model: 'all'
+          - dubbo-version: '3-1'
+            dubbo-versions: '3'
+            registry-model: 'instance'
+          - dubbo-version: '3-1'
+            dubbo-versions: '6'
+            registry-model: 'all'
+          - dubbo-version: '3-1'
+            dubbo-versions: '6'
+            registry-model: 'instance'
+          - dubbo-version: '3-1'
+            dubbo-versions: '9'
+            registry-model: 'all'
+          - dubbo-version: '3-1'
+            dubbo-versions: '9'
+            registry-model: 'instance'
+          - dubbo-version: '3-1'
+            dubbo-versions: '11'
+            registry-model: 'all'
+          - dubbo-version: '3-1'
+            dubbo-versions: '11'
+            registry-model: 'interface'
+          - dubbo-version: '3-1'
+            dubbo-versions: '11'
+            registry-model: 'instance'
+          - dubbo-version: '3-2'
+            dubbo-versions: '0'
+            registry-model: 'all'
+          - dubbo-version: '3-2'
+            dubbo-versions: '0'
+            registry-model: 'instance'
+          - dubbo-version: '3-2'
+            dubbo-versions: '3'
+            registry-model: 'all'
+          - dubbo-version: '3-2'
+            dubbo-versions: '3'
+            registry-model: 'instance'
+          - dubbo-version: '3-2'
+            dubbo-versions: '6'
+            registry-model: 'all'
+          - dubbo-version: '3-2'
+            dubbo-versions: '6'
+            registry-model: 'instance'
+          - dubbo-version: '3-2'
+            dubbo-versions: '11'
+            registry-model: 'all'
+          - dubbo-version: '3-2'
+            dubbo-versions: '11'
+            registry-model: 'interface'
+          - dubbo-version: '3-2'
+            dubbo-versions: '11'
+            registry-model: 'instance'
         exclude:
           - dubbo-version: '2-5'
-            dubbo-versions: '11'
+            dubbo-versions: '9'
+            registry-model: 'interface'
           - dubbo-version: '2-5'
             dubbo-versions: '12'
-          - dubbo-version: '2-7'
-            dubbo-versions: '1'
-          - dubbo-version: '2-7'
-            dubbo-versions: '2'
+            registry-model: 'interface'
           - dubbo-version: '2-7'
             dubbo-versions: '3'
-          - dubbo-version: '2-7'
-            dubbo-versions: '4'
+            registry-model: 'interface'
+          - dubbo-version: '3-1'
+            dubbo-versions: '12'
+            registry-model: 'interface'
+          - dubbo-version: '3-2'
+            dubbo-versions: '12'
+            registry-model: 'interface'
```

Comment:
3.0 3.1 3.2 run nothing for instance and all

Reply:
fixed